### PR TITLE
Varargs capitalization

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: framework/src/main/java/org/checkerframework/framework/util/AnnotatedTypes.java [hunk 1]
```diff
@@ -990,7 +990,7 @@ private static AnnotatedTypeMirror glbSubtype(
    *     Tree)} instead
    */
   @Deprecated // 2022-04-21
-  public static List<AnnotatedTypeMirror> expandVarArgsParameters(
+  public static List<AnnotatedTypeMirror> expandVarargsParameters(
```
Comment (smillst) [inline framework/src/main/java/org/checkerframework/framework/util/AnnotatedTypes.java:993]: Probably don't need to rename this deprecated method.

File: javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java [hunk 2]
```diff
@@ -2608,7 +2608,7 @@ public static boolean isVarargsCall(Tree tree) {
    * @deprecated use {@link #isVarargsCall(MethodInvocationTree)}
    */
   @Deprecated // 2024-06-04
-  public static boolean isVarArgs(MethodInvocationTree invok) {
+  public static boolean isVarargs(MethodInvocationTree invok) {
```
Comment (smillst) [inline javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java:2611]: Probably don't need to rename the deprecated method.

File: javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java [hunk 3]
```diff
@@ -2667,7 +2667,7 @@ public static boolean isCallToVarArgsMethodWithZeroVarargsActuals(MethodInvocati
    * @deprecated use {@link #isVarargsCall(NewClassTree)}
    */
   @Deprecated // 2024-06-04
-  public static boolean isVarArgs(NewClassTree newClassTree) {
+  public static boolean isVarargs(NewClassTree newClassTree) {
```
Comment (smillst) [inline javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java:2670]: Probably don't need to rename the deprecated method.

File: javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java [hunk 4]
```diff
@@ -2872,7 +2872,7 @@ public static boolean isStandaloneExpression(ExpressionTree expression) {
    * @deprecated use {@link #isVarargsCall(Tree)}
    */
   @Deprecated // 2024-06-04
-  public static boolean isVarArgMethodCall(ExpressionTree methodInvocation) {
+  public static boolean isVarargsMethodCall(ExpressionTree methodInvocation) {
```
Comment (smillst) [inline javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java:2875]: Probably don't need to rename the deprecated method.

**Review by smillst — APPROVED** [review]

**Comment by RTxin** [comment]
After pulling the code for this COMMIT on Windows，`EnsuresCalledMethodsVarargs.java` & `EnsuresCalledMethodsVarArgs.java`, recognised as the same file on Windows

**Comment by smillst** [comment]
@RTxin
> After pulling the code for this COMMIT on Windows，`EnsuresCalledMethodsVarargs.java` & `EnsuresCalledMethodsVarArgs.java`, recognised as the same file on Windows

Thanks for pointing this out.  I've opened [a pull request](https://github.com/typetools/checker-framework/pull/6720) to remove EnsuresCalledMethodsVarArgs.java.

**Pull request merged**